mne/io/eeglab/eeglab.py

```diff
@@ -354,6 +662,18 @@ def __init__(self, input_fname, montage, eog=(), event_id=None,
         self._create_event_ch(events, n_samples=eeg.pnts)
 
         # read the data
+        if isinstance(eeg, Bunch):
+            # if hdf5 data was read in, then file name might be encoded
+            # as array of ascii values of characters
+            ascii_check = _check_for_ascii_filename(eeg, input_fname)
+            if ascii_check[0]:
+                eeg.data = ascii_check[1]
+                hdf5_transpose = False
+            else:
+                hdf5_transpose = True
+        else:
+            hdf5_transpose = False
```

Comment:
It looks now like conversion is scattered -- is that correct? This for example could be moved to `_get_eeg_data` so that all conversion / unification takes place in one location.